app/javascript/legacy_react/src/components/common/DefaultCloseButton.tsx

```diff
@@ -64,10 +64,10 @@ const states: { [state: string]: any } = {
 @observer
 export class DefaultCloseButton extends React.Component<DefaultCloseButtonProps, {}> {
   @observable
-  hovering: boolean
+  hovering: boolean = false;
```

Comment:
Look into how the `@observable` decorator works. I don't believe this is the correct way to fix this.